pkg/printer/printer.go

```diff
@@ -191,25 +192,39 @@ func GetFlowType(f v1.Flow) string {
 
 // WriteProtoFlow writes v1.Flow into the output writer.
 func (p *Printer) WriteProtoFlow(res *observerpb.GetFlowsResponse) error {
+	var err error
+	write := func(w io.Writer, a ...interface{}) {
+		if err != nil {
+			return
+		}
+		_, err = fmt.Fprint(w, a...)
+	}
+
 	f := res.GetFlow()
+
 	switch p.opts.output {
 	case TabOutput:
+		var err error
```

Comment:
This is shadowing `err` declared at line 195. Because of that, the `err` check at line 234 will always take the false branch.